api/src/main/java/org/mandulis/mts/service/AttachmentValidationService.java

```diff
@@ -0,0 +1,25 @@
+package org.mandulis.mts.service;
+
+import lombok.RequiredArgsConstructor;
+import org.mandulis.mts.dto.AttachmentDto;
+import org.mandulis.mts.exception.AttachmentWithoutNameException;
+import org.mandulis.mts.exception.EmptyAttachmentException;
+import org.mandulis.mts.exception.TicketNotFoundException;
+import org.mandulis.mts.repository.TicketRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+@Service
+@RequiredArgsConstructor
```

Comment:
why do you need a required args constructor? There's nothing in the class to be injected 😅

Reply:
Mistake :D